pom.xml

```diff
@@ -289,5 +290,11 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>tez</id>
+            <properties>
+                <hive.execution.engine>tez</hive.execution.engine>
```

Comment:
We should configure the continuous build job to run the tez profile as well as the regular one.